kuryr_kubernetes/utils.py

```diff
@@ -319,7 +319,7 @@ def get_endpoints_link(service):
     link_parts = svc_link.split('/')
 
     if link_parts[-2] != 'services':
-        raise exceptions.IntegrityError(_(
```

Comment:
Don't we care about translations?

Reply:
Ah, that's cherrypick. Forget about the question ;)

Reply:
yeah, on master I saw we use f-strings https://github.com/openshift/kuryr-kubernetes/blob/master/kuryr_kubernetes/utils.py#L334-L335